securing_apps/topics/oidc/java/jboss-adapter.adoc

```diff
@@ -163,13 +163,13 @@ $ sudo subscription-manager repos --enable=jb-eap-6-for-rhel-<RHEL_VERSION>-serv
 Install the EAP 6 adapters for OIDC using the following command:
 
 ----
-$ sudo yum install eap6-keycloak-adapter
+$ sudo yum install eap6-keycloak-adapter-sso7_2-eap6
```

Comment:
Should be keycloak-adapter-sso7_2

Reply:
I just noticed the start of the adapter name still included "eap6-" here, but not below. Should it also be removed here?

Reply:
@matthewhelmke correct name is keycloak-adapter-sso7_2-eap6